### DISCUSSION TITLE
Single node install

### DISCUSSION THREAD

**Discussion by cmsj** [okd-project/okd#1658] [Q&A]
Hey folks!

I'm very interested in doing a single node install of OKD, but I'm struggling a bit with finding docs - there are several blog posts linked off okd.io but they seem very specific in their use cases and I was looking for something more general (e.g. I don't really want a whole separate network for this, just on my LAN).

Is there somewhere else I should be looking? Or some pointers people could share?

Many thanks!
Chris

**Comment by vrutkovs** [comment]
See https://github.com/okd-project/okd/discussions/1547 and https://upstreamwithoutapaddle.com/blog%20post/2023/03/06/Back-To-Where-It-All-Started.html

I wrote [a guide focusing on disconnected install](https://vrutkovs.eu/posts/okd-disconnected-assisted/), so feel free to create one focusing on single node installation.
